plugins/action/zos_job_submit.py

```diff
@@ -32,6 +35,14 @@ def run(self, tmp=None, task_vars=None):
             return result
 
         module_args = self._task.args.copy()
+
+        use_template = _process_boolean(module_args.get("use_template"))
+        if use_template and module_args.get("location") != "LOCAL":
+            result["failed"] = True
```

Comment:
If result is a dict consider using .update() function when updating multiple lines. Debate me on this, is just a comment about visuals. You can ignore this

Reply:
I don't think I need to debate you, I browsed other modules and all of them use `update` when updating multiple items in a dictionary. Thanks for bringing it up!